### PR TITLE
make the Governance page readable straight through

### PR DESCRIPTION
**Pull request by SethTisue**
this is a sequel to #1700

@tgodzik told me in conversation that he is open to a plain-text look instead of the cards

I think the cards made sense when they were at the top of https://www.scala-lang.org/community/ and it was important not to use too much vertical real estate

but now I think it's merely annoying to have click through all the cards in order to read everything

also, having everything in cards meant the page had no proper table of contents

in putting this PR together, I tried to avoid changing the text, but I did do some light editing/rearranging to reduce redundancy between the "Who's behind Scala?" sections and the other sections, to make the page make better sense as a single whole

### CODE REVIEW

**Comment by SethTisue** [comment]
the old look is currently still viewable at https://www.scala-lang.org/governance/

and here are screen shots showing the plain-text look I'm suggesting:

Screenshot 2024-10-21 at 1 00 02 PM
Screenshot 2024-10-21 at 1 00 06 PM

**Review by tgodzik — COMMENTED** [review]
Looks good! I don't have any particular preferences, but I guess plain text is easier to notice. Although, should we have a summary of all the links user might want to use? Isn't it too hidden in text now?

File: governance/index.md [hunk 1]
```diff
@@ -10,26 +10,63 @@ includeTOC: true
...
 at [EPFL](https://epfl.ch).
 
-The Scala language and associated websites are cooperative projects of the
-[Scala Center](https://scala.epfl.ch) at [EPFL](https://epfl.ch), the Scala 3
-team in Martin's research group, [LAMP](https://lamp.epfl.ch) (also at EPFL),
-the Scala teams at [Lightbend](https://www.lightbend.com) and
-[VirtusLab](https://virtuslab.com), and the Scala community more broadly, with
-participation from many companies, organizations, and individuals.
+The Scala language and associated websites are cooperative projects of these
+four groups:
```
Comment (Gedochao) [inline governance/index.md:14]: nitpick
```suggestion
The Scala language and associated websites are cooperative projects of the following
four groups:
```

**Review by Gedochao — APPROVED** [review]
LGTM

**Comment by tgodzik** [comment]
>  maybe you'd like to send a small followup PR on this...?

Maybe it's enough, though if at some point I have some unexpected strike of UX genius I will do a follow up

**Pull request merged**